src/module/linux/linux.h

```diff
@@ -1,9 +1,16 @@
 #pragma once
 // Linux specific definitions
 #include "syscalls.h"
+#include "netlink.h"
 
 #define KRF_SAFE_WRITE(x) KRF_CR0_WRITE_UNLOCK(x)
-#define KRF_LOG(...) printk(KERN_INFO __VA_ARGS__)
+#define KRF_LOG(...)                                                                               \
+  ({                                                                                               \
+    char buf[KRF_NETLINK_BUF_SIZE]; /* max message size */                                         \
+    printk(KERN_INFO __VA_ARGS__);                                                                 \
+    snprintf(buf, KRF_NETLINK_BUF_SIZE, __VA_ARGS__);                                              \
+    krf_netlink_broadcast(buf, strlen(buf) + 1);                                                   \
```

Comment:
We can probably save a `strlen` call here and use the return value of `snprintf` instead, since it'll return the number of bytes formatted.